sonar-core/src/main/java/org/sonar/core/issue/DefaultIssue.java

```diff
@@ -267,34 +267,55 @@ public DefaultIssue setLine(@Nullable Integer l) {
     return this;
   }
 
+  /**
+   * @deprecated since5.5, replaced by {@link #gap()}
+   */
+  @Deprecated
   @Override
   @CheckForNull
   public Double effortToFix() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)